lib/foreman_maintain/cli/base.rb

```diff
@@ -101,7 +103,9 @@ def self.interactive_option
         delete_duplicate_assumeyes_if_any
 
         option ['-y', '--assumeyes'], :flag,
-               'Automatically answer yes for all questions'
+               'Automatically answer yes for all questions' do |assume|
+          ForemanMaintain.reporter.assumeyes(assume)
```

Comment:
@ntkathole, 
Thanks for testing!
Error that you specified is due to this line. 
Instead of method call, assignment for `assumeyes`  needed here.